counterpoll/main.py

```diff
@@ -203,7 +203,7 @@ def watermark():
     """ Watermark counter commands """
 
 @watermark.command()
-@click.argument('poll_interval', type=click.IntRange(1000, 30000))
+@click.argument('poll_interval', type=click.IntRange(1000, 60000))
```

Comment:
this is the only interval that needs to be updated. Please remove the rest of the changes. They are not related to watermark counters